react/Popover.tsx

```diff
@@ -34,27 +41,25 @@ function useTransition(option: TransitionComponentType) {
   }, [option])
 }
 
-const CSS_HANDLES = ['container']
+const CSS_HANDLES = ['container', 'arrow']
```

Comment:
```suggestion
const CSS_HANDLES = ['container', 'arrow'] as const
```

Reply:
That doesn't change anything since we don't type the `useCssHandles` right